downstream/modules/eda/proc-eda-activate-webhook.adoc

```diff
@@ -0,0 +1,73 @@
+[id="proc-eda-activate-webhook"]
+
+= Activating webhook rulebooks
+
+Rulebooks that include a "source" section require extra configuration to accept external messages as triggers, usually in the form of a webhook. 
+In Openshift environments, you can allow webhooks to reach an activation-job-pod over a given port by creating a Route that exposes that rulebook activation's K8s service.
```

Comment:
```suggestion
In Openshift environments, you can allow webhooks to reach an activation-job-pod over a given port by creating a Route that exposes that rulebook activation's K8s service.
```
I don't know whether we can use the K8s abbreviation in docs, or whether we have to use Kubernetes